packages/typescript-estree/src/create-program/createDefaultProgram.ts

```diff
@@ -32,7 +32,10 @@ function createDefaultProgram(
   const commandLine = ts.getParsedCommandLineOfConfigFile(
     tsconfigPath,
     DEFAULT_COMPILER_OPTIONS,
-    { ...ts.sys, onUnRecoverableConfigFileDiagnostic: () => {} },
+    {
+      ...ts.sys,
+      onUnRecoverableConfigFileDiagnostic: () => {}, // eslint-disable-line @typescript-eslint/no-empty-function
```

Comment:
instead of disabling this line here and in `createWatchProgram.ts`,
could you please change the eslint config to allow empty arrow functions?
`allow: ["arrowFunctions"]`

Reply:
Sure, I thought about that option ;)